pallets/creditcoin/src/lib.rs

```diff
@@ -1170,6 +1170,8 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			transfer: Transfer<T::AccountId, T::BlockNumber, T::Hash, T::Moment>,
 		) -> DispatchResultWithPostInfo {
+			let tx_hash = hex::encode(&transfer.tx_id);
+			log::info!("**** DEBUG, verify_transfer, tx_id={:?}", tx_hash);
```

Comment:
From `creditcoin-node.log` we can see the following
```
2022-04-25 21:03:34 **** DEBUG, verify_transfer, tx_id="ca78ecd0a93b08ac2dda0fed6a0399990cf456230574d80fd8aa0507a401fcf7"    
2022-04-25 21:04:05 **** DEBUG, verify_transfer, tx_id="c79d9bc1d1d928ea3f2cade432b81ffed7c1776d33991bc771f2d17a03977e25"    
2022-04-25 21:04:54 **** DEBUG, verify_transfer, tx_id="98f87aa080de9933698030d3b08f78778b3728ee928668fb8659c5ed777f460c"    
2022-04-25 21:05:24 **** DEBUG, verify_transfer, tx_id="8b739fc155fa9507f03e4bef2412f0bbfafbe127919e3ed36206d07887a9d5ab"    
2022-04-25 21:06:00 **** DEBUG, verify_transfer, tx_id="0826d571a4fe8b6dc54162f9a30d6a2375dfd0b39b56583776a21a4bc5d6b10e"    
2022-04-25 21:06:00 **** DEBUG, verify_transfer, tx_id="0826d571a4fe8b6dc54162f9a30d6a2375dfd0b39b56583776a21a4bc5d6b10e"    
2022-04-25 21:07:57 **** DEBUG, verify_transfer, tx_id="b5f67ef7d98996a4426895385c8405fd64a6693bf55ac32f1464112b7b1129f1"    
2022-04-25 21:08:43 **** DEBUG, verify_transfer, tx_id="da1cd3a30177c54249fe1dc145dd16d81a37556d00295883165d39b594b24e60"    
2022-04-25 21:09:39 **** DEBUG, verify_transfer, tx_id="cb117d8b63175abe7c9da1af0df31e2432c1937152a24d1c80222ef44ec07cae"   
```

Where verify_transfer is called twice with the `0x0826d571a4fe8b6dc54162f9a30d6a2375dfd0b39b56583776a21a4bc5d6b10e` tx_id. Which is the same ID that causes the VerifyTransfer spec to fail.